datalab/datalab.sh

```diff
@@ -16,7 +16,17 @@
 # This init script installs Google Cloud Datalab on the master node of a
 # Dataproc cluster.
 
-set -e -x
+set -exo pipefail
```

Comment:
Other scripts seem to be using `set -euxo pipefail`

Reply:
`-u` is not possible - this script uses environment variables from the system that are not declared in this script